lib/web/apiserver.go

```diff
@@ -110,7 +120,14 @@ type Handler struct {
 	sessionStreamPollPeriod time.Duration
 	clock                   clockwork.Clock
 	limiter                 *limiter.RateLimiter
-	healthCheckAppServer    healthCheckAppServerFunc
+	highLimiter             *limiter.RateLimiter
```

Comment:
This is unrelated to the assistant limiter and should not be backported.

Reply:
Removed